_linter/markdown-linter-style.rb

```diff
@@ -0,0 +1,33 @@
+#  Include all rules by default
+all
+
+#  Rule Modifications
+#  --------------------------------
+## MD029 mod - tells linter not to fail if order list items don't start with 1
```

Comment:
This seems like a rule we'd want to follow. Do we have ordered lists that don't begin with 1?